hoomd/md/TwoStepNPTMTK.cc

```diff
@@ -981,18 +981,13 @@ TwoStepNPTMTK::couplingMode TwoStepNPTMTK::getRelevantCouplings()
     return couple;
     }
 
-void TwoStepNPTMTK::randomizeVelocities(unsigned int timestep)
+void TwoStepNPTMTK::thermalizeExtraDOF(unsigned int seed, unsigned int timestep)
     {
-    if (m_shouldRandomize == false)
-        {
-        return;
-        }
-
-    m_exec_conf->msg->notice(6) << "TwoStepNPTMTK randomizing velocities" << std::endl;
+    m_exec_conf->msg->notice(6) << "TwoStepNPTMTK randomizing extra DOF" << std::endl;
```

Comment:
I find the `extra` naming to be confusing, I have to read the class or the documentation in the header to recognize which DOF are under consideration. Could we instead name it to something like `thermalizeReservoirDOF`? I'm open to other names as well, I realize that my suggestion might be too broad if there are reservoirs (e.g. Gibbs ensemble simulations) where that's not quite right either, but I'd like the name to more clearly indicate what DOF are being considered here.

Reply:
I moved the introduction of the thermostat and barostat degrees of freedom up into the main NVT and NPT documentation, and renamed the methods.